addon/components/nrg-datetime/calendar.js

```diff
@@ -241,47 +240,54 @@ export default Component.extend(EKMixin, EKFirstResponderOnFocusMixin, {
     }
 
     return rows;
-  }),
-
-  table: computed(
-    'isSelectingDays',
-    'isSelectingMonths',
-    'isSelectingYears',
-    'isSelectingHours',
-    'isSelectingMinutes',
-    'days',
-    'months',
-    'years',
-    'hours',
-    'minutes',
-    function() {
-      let columnHeaders = DAY_HEADERS;
-      let columnCountClass = 'seven';
-      let rows = this.days;
-      if (this.isSelectingMonths) {
-        columnHeaders = null;
-        columnCountClass = 'three';
-        rows = this.months;
-      } else if (this.isSelectingYears) {
-        columnHeaders = null;
-        columnCountClass = 'two';
-        rows = this.years;
-      } else if (this.isSelectingHours) {
-        columnHeaders = null;
-        columnCountClass = 'four';
-        rows = this.hours;
-      } else if (this.isSelectingMinutes) {
-        columnHeaders = null;
-        columnCountClass = 'three';
-        rows = this.minutes;
-      }
-      return {
-        columnCountClass,
-        columnHeaders,
-        rows,
-      };
+  }
+
+  get table() {
+    let columnHeaders = DAY_HEADERS;
+    let columnCountClass = 'seven';
+    let rows = this.days;
+    if (this.isSelectingMonths) {
+      columnHeaders = null;
+      columnCountClass = 'three';
+      rows = this.months;
+    } else if (this.isSelectingYears) {
+      columnHeaders = null;
+      columnCountClass = 'two';
+      rows = this.years;
+    } else if (this.isSelectingHours) {
+      columnHeaders = null;
+      columnCountClass = 'four';
+      rows = this.hours;
+    } else if (this.isSelectingMinutes) {
+      columnHeaders = null;
+      columnCountClass = 'three';
+      rows = this.minutes;
+    }
+    return {
+      columnCountClass,
+      columnHeaders,
+      rows,
+    };
+  }
+
+  isBeyondDateRange(date, precision) {
+    date = moment(date);
+    let valid = true;
+    if (this.args.minDate) {
+      valid = valid && !date.isBefore(this.args.minDate, precision);
```

Comment:
[LGTM doesn't like this line](https://lgtm.com/projects/g/knoxville-utilities-board/ember-nrg-ui/snapshot/6af3cef033927bf9f1d3b1350d9f51f9ee313292/files/addon/components/nrg-datetime/calendar.js?sort=name&dir=ASC&mode=heatmap#xa63bf1fc811006ce:1), since `valid` is defined to be true right above this
```suggestion
      valid = !date.isBefore(this.args.minDate, precision);
```